pkgs/by-name/li/linux-wallpaperengine/package.nix

```diff
@@ -28,41 +28,50 @@
   cef-binary,
   libdecor,
   autoPatchelfHook,
+  python3,
+  libpng,
+  fftw,
+  file,
+  kissfftFloat,
 }:
 
 let
   cef = cef-binary.overrideAttrs (oldAttrs: {
-    version = "120.1.10";
+    version = "135.0.17";
     __intentionallyOverridingVersion = true; # `cef-binary` uses the overridden `srcHash` values in its source FOD
-    gitRevision = "3ce3184";
-    chromiumVersion = "120.0.6099.129";
+    gitRevision = "cbc1c5b";
+    chromiumVersion = "135.0.7049.52";
```

Comment:
Maybe it's worth it if we put a comment here explaining that this information needs to [follow upstream](https://github.com/Almamu/linux-wallpaperengine/blob/be0fc25e72203310f268221a132c5d765874b02c/CMakeLists.txt#L47)